### ISSUE TITLE
Add the ability to change buttons showed on keys

### ISSUE THREAD

**Issue by ifohancroft** [hishamhm/htop#887]
Add the ability, to be able to change whether the F1-F10 keys will be shown on the buttons or the secondary keys like h,S,\,/,etc.
Ideally, per user basis.

**Comment by Explorer09** [comment]
Why? Is there any case where this feature can be useful?

**Comment by ifohancroft** [comment]
People using 60% keyboards (those without the F row and that have nothing to the right from Shift) like myself. You can press stuff like F12 by pressing Fn + = or Fn+4 for F4 but since htop has the ability to use secondary keys anyway, it will be useful to make it show the keys we'll actually be using instead of the F ones. Also, by seeing them we can actually start remembering them. As of right now, I would rather use the secondary keys as it's tedious to press to keys when you can press one but I can't for the life of me remember the secondary keys of htop. Having them show on the buttons will help that.

**Comment by Explorer09** [comment]
＠ifohancroft So this feature request is about the _display_ of the "FunctionBar" only, and not about the key remapping of the functions?

**Comment by ifohancroft** [comment]
@Explorer09 Yes. Only about the display of the "FunctionBar". I am sorry if I failed to explain it properly and caused confusion about it.
